src/main/java/attendance/domain/AttendanceBook.java

```diff
@@ -0,0 +1,87 @@
+package attendance.domain;
+
+import attendance.util.ErrorMessage;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class AttendanceBook {
+
+    private final Map<Crew, Attendances> attendanceBook;
+
+    public AttendanceBook(Map<Crew, Attendances> attendanceBook) {
+        this.attendanceBook = attendanceBook;
+    }
+
+    public void add(Crew crew, Attendance attendance) {
+        if (attendanceBook.containsKey(crew)) {
+            attendanceBook.get(crew).add(attendance);
+            return;
+        }
+
+        attendanceBook.put(crew, new Attendances(new ArrayList<>(List.of(attendance))));
+    }
+
+    public Attendance findAttendanceByCrew(Crew crew, LocalDate inputDate) {
+        return attendanceBook.get(crew).findByDate(inputDate);
+    }
```

Comment:
```suggestion
    public Attendance findAttendanceByCrewAndDate(Crew crew, LocalDate inputDate) {
        Attendances attendances = attendanceBook.get(crew);
        return attendances.findByDate(inputDate);
    }
```
저라면 가독성을 위해 이렇게 할 것 같아요~